### PR TITLE
MINOR: Fix typos in project

### PR DESCRIPTION
**Pull request by guiyanakuang**
### What changes were proposed in this pull request?

This PR aims to fix a typos in project.

### Why are the changes needed?

Easy to read

### How was this patch tested?

Pass the CIs.

### CODE REVIEW

File: java/bench/core/src/java/org/apache/orc/bench/core/CompressionKind.java [hunk 1]
```diff
@@ -36,8 +36,8 @@
   SNAPPY("snappy"),
   ZSTD("zstd");
 
-  CompressionKind(String extendsion) {
```
Comment (dongjoon-hyun) [inline java/bench/core/src/java/org/apache/orc/bench/core/CompressionKind.java:39]: Nice catch. :)

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.
Merged to main/1.7/1.6

File: java/bench/core/src/java/org/apache/orc/bench/core/CompressionKind.java (hunk 1 above)
Comment (guiyanakuang) [reply java/bench/core/src/java/org/apache/orc/bench/core/CompressionKind.java:39]: I found a few, but most of them were scanned with tools. :blush:

**Comment by dongjoon-hyun** [comment]
I'll cherry-pick this to branch-1.5, too.

**Pull request merged**